docs/source/gpu.rst

```diff
@@ -25,29 +49,44 @@ As a worked example, you may want to view this talk:
                 type="video/mp4">
     </video>
 
-High Level Collections
-----------------------
+`Dask and V100s for Fast, Distributed Batch Scoring of Computer Vision Workloads <https://developer.nvidia.com/gtc/2019/video/s9198>`_
+
+Having parallel tasks use GPUs
+------------------------------
+
+Tasks like running experiments of different parameters of a machine learning model on a GPU
+can be extremely parallelized. Dask can be used to run each experiment trial concurrently,
```

Comment:
```suggestion
can are embarrassingly parallelizable. Dask can be used to run each experiment trial concurrently,
```

Reply:
So I think it previously said embarrassingly but I took it out because that seemed overly negative. Do you think embarrassingly won't like turn away new people?